spec/unit/gzip_spec.rb

```diff
@@ -92,6 +95,19 @@
       it_behaves_like 'compressed response'
     end unless jruby?
 
+    context 'empty response' do
+      let(:body) { empty_body }
+      let(:headers) { {'Content-Encoding' => 'gzip', 'Content-Length' => body.length } }
+
+      it 'sets the Content-Length' do
+        expect(process(body).headers['Content-Length']).to eq(empty_body.length)
+      end
+
+      it 'removes the Content-Encoding' do
+        expect(process(body).headers['Content-Encoding']).to be_nil
+      end
+    end unless jruby?
```

Comment:
This should be tested and work under `jruby` as well

Reply:
No problem. Yeah, it seems like a simple case, but I guess it doesn't happen that often.
I have removed the exception for jruby in the test now.